packages/generator/src/v1.1/generateMetadataFile.ts

```diff
@@ -32,7 +32,10 @@ export async function generateMetadataFile(
   import type { OntologyDefinition } from "@osdk/api";
   import type { Ontology as ClientOntology } from "@osdk/legacy-client";
   import type { Objects } from "./ontologyObjects";
-  ${objectNames.map((name) => `import {${name}} from "./objects/${name}";`)}
+  ${
+      objectNames.map((name) => `import {${name}} from "./objects/${name}";`)
+        .join("\n")
```

Comment:
We need to remember to do this. newline on join or we end up with "," that we dont want in the code